matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt

```diff
@@ -57,7 +58,7 @@ internal class RealmSessionStoreMigration @Inject constructor(
     override fun equals(other: Any?) = other is RealmSessionStoreMigration
     override fun hashCode() = 1000
 
-    val schemaVersion = 24L
+    val schemaVersion = 25L
```

Comment:
The value you want to set is 26, there was a mistake, the previous upgrade did not increment the version.

Reply:
Thanks yes, I already fixed that in the` live thread list` that will also contain this PR